focus-android/.buildconfig.yml

```diff
@@ -55,6 +55,7 @@ projects:
     - support-base
     - support-images
     - support-ktx
+    - support-license
```

Comment:
@jonalmeida Hey! I think you can fix the issues called out in [bug 1826122](https://bugzilla.mozilla.org/show_bug.cgi?id=1826122) by adding the same line to `fenix/.buildconfig.yml`

Reply:
This component isn't used in fenix though, only in Focus right now.